pycqed/measurement/pulse_sequences/single_qubit_tek_seq_elts.py

```diff
@@ -833,6 +834,35 @@ def over_under_rotation_seq(qb_name, nr_pi_pulses_array, operation_dict,
     return
 
 
+def res_amp_sweep_seqs(qb_name, hard_sweep_dict, soft_sweep_dict,
```

Comment:
This function is really cryptic to me.

Reply:
Is it some general readout parameter 2d sweep sequence?

Reply:
@nathlacroix ?

Reply:
same for me, A doc string might be helpful, as I have commented in my review

Reply:
The measurement corresponding to this sequence doesn't work anyway. I should have removed it. I will do so now, along with the measure function.